.env

```diff
@@ -5,7 +5,7 @@ STUDIO_CONFIG_PACKAGE=picknik_ur_mock_hw_config
 
 # MoveIt Studio version tag
 # Be careful! Changing this without understanding the consequences may break your installation. 
-STUDIO_DOCKER_TAG=2.11.0
+STUDIO_DOCKER_TAG=main
```

Comment:
So... this `.env` file is actually used for https://docs.picknik.ai/en/stable/getting_started/configuration_tutorials/docker_configuration/create_overlay_docker_image.html, and should therefore mostly be used with a release tag here.

Is the idea that on the `main` branch this will always be `main`, but in each of the release-specific branches, like `v3.0` we would use `3.0.0` there?

Reply:
Yes. The main branches will always be main, but release specific branches this should point to the version that went through the release process and was tested...

This way if you back port something to a released branch the `.env` file on that release branch should have the tag set correctly and it will test against the correct version of studio.

Reply:
The tutorials get the value from [here](https://github.com/PickNikRobotics/moveit_studio/blob/main/docs_public/source/getting_started/configuration_tutorials/docker_configuration/_include/overlay_docker_env.rst) not this file specifically so if someone is following along and copy-pasting from the tutorials they should be fine.

Reply:
Thanks for the answer, and good point on the tutorial source being more likely to be copypasted than someone trying it out directly from here. LGTM.